src/containers/FormikForm/formikDraftHooks.ts

```diff
@@ -20,6 +20,23 @@ import {
 import { useDraftHistory } from "../../modules/draft/draftQueries";
 import { useTaxonomyVersion } from "../StructureVersion/TaxonomyVersionProvider";
 
+const updateUserData = async (articleId: number) => {
+  const stringId = articleId.toString();
+  const result = await fetchUserData();
+  const latest = uniq([stringId].concat(result.latestEditedArticles ?? []));
+  const latestEditedArticles = latest.slice(0, LAST_UPDATED_SIZE);
+  apiUpdateUserData({ latestEditedArticles });
+};
+
+const checkArticleChanged = (old: IArticle | undefined, upd: IUpdatedArticle): boolean => {
+  if (!old) throw new Error("Did not get old article when checking for changes");
+  return (
+    (old.title?.title ?? "") !== (upd.title ?? "") ||
+    (old.introduction?.introduction ?? "") !== (upd.introduction ?? "") ||
+    (old.content?.content ?? "") !== (upd.content ?? "")
```

Comment:
Burde vi differensiere mellom "" og undefined her?

Reply:
Vil ikke det kreve endringer?

Sånn i praksis så tror jeg ikke det har stort å si uansett, men vi lagrer tydeligvis `introduction` som tom-streng uansett om den er tom.

Så tror vi vil få en del falske positiver dersom vi ikke også endrer på hva vi sender til api'et ved lagring 🤷

Reply:
Klarer du egentlig å komme på noen situasjoner hvor det har noe å si? For jeg tror ikke jeg klarer sånn ved første øyekast 😅